.drone.star

```diff
@@ -452,6 +452,14 @@ def main(ctx):
         ),
     )
 
+    if "coverage" in ctx.build.title.lower():
```

Comment:
why only run them if s.o. puts `coverage` in the title? Do CI runs with coverage take longer?
Same question for the cron job runs.
If we run them on every PR or at least merge, we could display batches in the README.md